templates/bridgeserver2.yaml

```diff
@@ -137,6 +152,57 @@ Parameters:
 Conditions:
   CreateDevResources: !Equals [ !Ref BridgeEnv, dev ]
 Resources:
+  DBCluster:
+    Type: AWS::RDS::DBCluster
+    Properties:
+      AvailabilityZones: !Ref DatabaseClusterAvailabilityZones
+      BackupRetentionPeriod: !Ref DatabaseBackupRetentionDays
+      DatabaseName: 'BridgeDB'
```

Comment:
Shouldn't this be ref to `DBInstance`?

Reply:
This refers to the database name in MySQL, not the RDS instance. We used BridgeDB for the manually created Aurora instance, and we're migrating those tables from the same Aurora instance, so I think we should be consistent with the naming.